tests/features/payments.feature

```diff
@@ -48,79 +39,68 @@ Scenario: Update a payment (PUT)
-    When I try to delete a non-existent payment with id 100
-    Then I should be returned nothing for payment 100
-
 Scenario: Query payments (Single Query)
     When I query payments with a valid query "payment_type" = "credit"
     Then I should see "1" existing payments
-    And I should see a payment with id "1" and "nickname" = "my credit"
+    And I should see a payment with id "1" and "nickname" = "cashmoney"
     When I query payments with a bad query "nickname" = "amex"
-    Then I should see an error message saying "Requested resource(s) could not be found" with status code "404"
+    Then I should see an error message saying "Requested resource(s) could not be found"
     
 Scenario: List all payments
     When I list all "payments"
-    Then I should see "3" existing payments
-    And I should see payment "1" with id "1" and "nickname" = "my credit"
-    And I should see payment "2" with id "2" and "nickname" = "my debit"
-    And I should see payment "3" with id "3" and "nickname" = "my paypal"
-    When I try to delete payment 1
-    When I try to delete payment 2
-    When I try to delete payment 3
-    When I list all "payments" when no payments exist
-    Then I should see "Requested resource(s) could not be found" with status code "404"
-
-Scenario: Use two query parameters to list payments
-    When I query "payments" with "nickname" = "my credit" and "payment_type" = "credit"
     Then I should see "1" existing payments
-    And I should see a payment with "nickname" = "my credit" and "payment_type" = "credit"
-    When I query "payments" with "ids" = "1" and "ids" = "2"
-    Then I should see "2" existing payments
-    And I should see payment "1" with id "1" and "nickname" = "my credit"
-    And I should see payment "2" with id "2" and "nickname" = "my debit"
-    When I query "payments" with "ids" = "1" and "payment_type" = "debit"
-    Then I should see "1" existing payments
-    And I should see a payment with id "1" and "nickname" = "my credit"
-    And I should not see "debit"
-    When I query "payments" with bad query inputs "nickname" = "my express card" and "payment_type" = "American Express"
-    Then I should see "Requested resource(s) could not be found" with status code "404"
+    And I should see payment "1" with id "1" and "nickname" = "cashmoney"
 
 Scenario: Read a payment
     When I get "payments" with id "1"
-    Then I should see a payment with id "1" and "nickname" = "my credit"
+    Then I should see a payment with id "1" and "nickname" = "cashmoney"
     And I should not see "Requested resource(s) could not be found"
     When I retrieve "payments" with non-existent id "100"
-    Then I should see "Payment with id 100 could not be found" with status code "404"
+    Then I should see "Payment with id 100 could not be found"
+
+Scenario: Use two query parameters to list payments
+    When I query "payments" with "nickname" = "cashmoney" and "payment_type" = "credit"
```

Comment:
It's a small change, but we could make this be `When I query my payments ...`.  The method logic would need to be changed, since we'd have to direct it to use the `payments` endpoint specifically.